manifests/init.pp

```diff
@@ -29,8 +29,10 @@
   $tomcat = $gutterball::params::tomcat,
 ) inherits gutterball::params {
 
+  $_ = cache_data('gutterball_db_password', $dbpassword)
+  $__ = cache_data('gutterball_db_user', $dbuser)
+  $___ = cache_data('gutterball_tomcat', $tomcat)
```

Comment:
Unfortunately, you have to take this in context with -- https://github.com/Katello/katello-installer/pull/201/files  whereby 'params' need to be accessed that aren't top-level parameters that Kafo would naturally have access to. We've (@stbenjam and myself) chatted about about doing the migrations and upgrade in the puppet modules directly due to things like this. If you had any thoughts around this design wise, we'd love to hear them :)